assignments/lambda-classes.js

```diff
@@ -1 +1,117 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(attributes) {
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+    this.gender =attributes.gender;
+}
+speak() {
+  return `Hello my name is ${this.name}, I am from ${this.location}`;
+}
+}
+
+class Instructor extends Person {
+    constructor(childAttributes) {
+        super(childAttributes);
+        this.specialty = childAttributes.specialty;
+        this.favLanguage = childAttributes.favLanguage;
+        this.catchPhrase = childAttributes.catchPhrase;
+    }
+    demo(subject) {
+        return `Today we are learning about ${subject}`;
+    }
+    grade(student, subject) {
+        return `${student.name} receives a perfect score on ${subject}`;
+    }
+}
+
+
+class Student extends Person {
+    constructor(gChildAttributes) {
```

Comment:
I'm going to leave that first comment, but I see what you were trying to do here now. You should also note that this is not a grandchild because it is 1 deep. For cases like this one, I would name all of the parameters the same.